### ISSUE TITLE
Don't block the opening of an existing wallet that uses a local backend even when blockchain syncing ongoing

### ISSUE THREAD

**Issue by nondiremanuel** [wizardsardine/liana#1384]
An activity related to #1377 would be to allow a user that has a Liana wallet already installed with local backend to not need to wait the blockchain syncing to open it. This is particularly relevant for the case of Liana managed bitcoind.
A practical usecase would be one that we encountered recently: if a user needs to urgently receive funds and he didn't open the wallet for a while, he should not have to wait for the blockchain synchronization to be able to generate an address. He should be able to open the wallet while being aware of the fact that the wallet is still syncing and that the information could be not up-to-date.

Since the solution for #1377 was to show the "syncing" state until the first poll runs and since the first poll typically runs only after the blockchain synchronization*, as a first step it should be enough to just eliminate the syncing step in the GUI and directly open the wallet while still performing the sync in background. The fact that the wallet is syncing is already communicated to the user by the syncing state implemented in the above PR. 

A second step of this work (which is not part of this issue), could be to improve the information communicated to the user by showing the progress on the blockchain syncing if this is happening, while keeping it generic for the cases in which it is synced and we are waiting for the wallet to catch up (i.e. the poller has not run yet).

For the moment, I would not apply this change to the IBD situation, i.e. if the user never opened the wallet before (checked via the wallet height). This is because the progress bar step after the backend choice seems to be an important moment for the user to realize the time and space implications of using a managed bitcoind and could affect its choice of using other more immediate solutions such as Liana Connect. This is valid also considering that we don't currently give the possibility of switching to a different backend after opening the wallet (other than recreating it). This point is open to discussion and could be also targeted later, when we will have a proper communication of the progress of the syncing also in the wallet home.

*The only corner case in which this is not the case seems to be when the user broadcasts a transaction while the blockchain is still syncing. This automatically triggers a poll and would make the syncing state disappear. As per the discussion we had with @jp1ac4, we could manage such case by not automatically performing the poll after broadcasting a transaction if the blockchain sync is still ongoing, which seems a consistent behavior also in general.

**Comment by jp1ac4** [comment]
> *The only corner case in which this is not the case seems to be when the user broadcasts a transaction while the blockchain is still syncing. This automatically triggers a poll and would make the syncing state disappear. As per the discussion we had with @jp1ac4, we could manage such case by not automatically performing the poll after broadcasting a transaction if the blockchain sync is still ongoing, which seems a consistent behavior also in general.

I've added a commit to https://github.com/wizardsardine/liana/pull/1376 to not poll after user broadcasts a spend if blockchain is syncing.

**Comment by kloaec** [comment]
>For the moment, I would not apply this change to the IBD situation

Seems fair. A user might want to deposit their coins to the wallet though, but maybe this is a edge case itself.
Example: I set up a good Liana multisig, I store my keys securely, and I want to move my coins before the IBD completes.
(i'm only going to save a couple of days)

**Comment by jp1ac4** [comment]
Another option could be to open the wallet once the the IBD progress reaches X%.

**Comment by nondiremanuel** [comment]
> > For the moment, I would not apply this change to the IBD situation
> 
> Seems fair. A user might want to deposit their coins to the wallet though, but maybe this is a edge case itself. Example: I set up a good Liana multisig, I store my keys securely, and I want to move my coins before the IBD completes. (i'm only going to save a couple of days)

I personally believe that, based on how we show the information right now, the risk of skipping to the home screen of the wallet during IBD without understanding that it will require a lot of time and disk space is higher than the benefit of this edge case. We also offer a good workaround with Liana Connect (or Electrum). 
I think we should reconsider this when we will show more details on the blockchain syncing.
